File: scent-quest-ui/src/app/services/models/review-fragrance-response.ts
/* tslint:disable */
/* eslint-disable */
import { UserResponse } from '../models/user-response';
export interface ReviewFragranceResponse {
  createdBy?: number;
  createdDate?: string;
  creator?: UserResponse;
  id?: number;
  modifiedAt?: string;
  modifiedBy?: number;
  rating?: number;
  text?: string;
}

File: scent-quest-api/src/main/java/com/kun/scentquest/feedback/ReviewFragranceResponse.java
package com.kun.scentquest.feedback;
import com.kun.scentquest.user.User;
import com.kun.scentquest.user.UserResponse;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ReviewFragranceResponse {

    private Integer id;
    private String text;
    private Integer rating;
    private LocalDateTime createdDate;
    private Integer createdBy;
    private Integer modifiedBy;
    private LocalDateTime modifiedAt;
    private UserResponse creator;

}
